c/parallel/src/kernels/iterators.cpp

```diff
@@ -161,7 +170,7 @@ struct __align__({5}) output_iterator_state_t{{
 struct output_iterator_t {{
   using iterator_category = cuda::std::random_access_iterator_tag;
   using difference_type   = {0};
-  using value_type        = void;
+  using value_type        = VALUE_T;
```

Comment:
Is this change required independent of https://github.com/NVIDIA/cccl/issues/3722?

If not, I'd recommend we make all the output_iterator changes in that PR.